src/apps/bm_devkit/sdi12_example/user_code/user_code.cpp

```diff
@@ -57,7 +57,7 @@ void transmit_samples(){
   tx_hex[i * 2] = '\0'; // Null-terminate the string
 
   // print to console
-  printf("[exo3-sonde-sdi12] buffer to send | tick: %llu, rtc: %s, buff: %s\n", uptimeGetMs(), rtcTimeBuffer, tx_hex);
+//  printf("[exo3-sonde-sdi12] buffer to send | tick: %llu, rtc: %s, buff: %s\n", uptimeGetMs(), rtcTimeBuffer, tx_hex);
```

Comment:
Was this intentional?

Reply:
Yes! It is. I shall delete it to remove confusion